sycl/doc/GetStartedGuide.md

```diff
@@ -5,36 +5,38 @@ and a wide range of compute accelerators such as GPU and FPGA.
 
 ## Table of contents
 
-* [Prerequisites](#prerequisites)
-  * [Create DPC++ workspace](#create-dpc-workspace)
-* [Build DPC++ toolchain](#build-dpc-toolchain)
-  * [Build DPC++ toolchain with libc++ library](#build-dpc-toolchain-with-libc-library)
-  * [Build DPC++ toolchain with support for NVIDIA CUDA](#build-dpc-toolchain-with-support-for-nvidia-cuda)
-  * [Build DPC++ toolchain with support for HIP AMD](#build-dpc-toolchain-with-support-for-hip-amd)
-  * [Build DPC++ toolchain with support for HIP NVIDIA](#build-dpc-toolchain-with-support-for-hip-nvidia)
-  * [Build DPC++ toolchain with support for ESIMD CPU Emulation](#build-dpc-toolchain-with-support-for-esimd-cpu-emulation)
-  * [Build DPC++ toolchain with support for runtime kernel fusion](#build-dpc-toolchain-with-support-for-runtime-kernel-fusion)
-  * [Build Doxygen documentation](#build-doxygen-documentation)
-  * [Deployment](#deployment)
-* [Use DPC++ toolchain](#use-dpc-toolchain)
-  * [Install low level runtime](#install-low-level-runtime)
-  * [Obtain prerequisites for ahead of time (AOT) compilation](#obtain-prerequisites-for-ahead-of-time-aot-compilation)
-    * [GPU](#gpu)
-    * [CPU](#cpu)
-    * [Accelerator](#accelerator)
-  * [Test DPC++ toolchain](#test-dpc-toolchain)
-    * [Run in-tree LIT tests](#run-in-tree-lit-tests)
-    * [Run DPC++ E2E test suite](#run-dpc-e2e-test-suite)
-    * [Run Khronos\* SYCL\* conformance test suite (optional)](#run-khronos-sycl-conformance-test-suite-optional)
-  * [Run simple DPC++ application](#run-simple-dpc-application)
-  * [Build DPC++ application with CMake](#build-dpc-application-with-cmake)
-  * [Code the program for a specific GPU](#code-the-program-for-a-specific-gpu)
-  * [Using the DPC++ toolchain on CUDA platforms](#using-the-dpc-toolchain-on-cuda-platforms)
-* [C++ standard](#c-standard)
-* [Known Issues and Limitations](#known-issues-and-limitations)
-  * [CUDA back-end limitations](#cuda-back-end-limitations)
-  * [HIP back-end limitations](#hip-back-end-limitations)
-* [Find More](#find-more)
+- [Getting Started with oneAPI DPC++](#getting-started-with-oneapi-dpc)
+  - [Table of contents](#table-of-contents)
```

Comment:
I'm not sure if these two items are useful. I guess we intentionally skipped them.

What is the reason for changing `*` with `-`?

Reply:
Makes sense, I've removed them.

I didn't notice the * -> - change, I think the lint did this. I've changed it back.